### ISSUE TITLE
Moderate gore option

### ISSUE THREAD

**Issue by Vairoon** [studio-minus/ppg-bugs-and-requests#1761]
**Feature description**
Instead of just "Goreless enabled/disabled" there should be "Gore All/Moderate/None". All is the same as deactivated and none is the same as enabled. The new one here is then moderate, and within that there should be:
Visible wounds, but not as bloody. Blood, but less bad. Maybe more, but they are some of the ones I thing would be needed for that option.

Alternatively, an option for wound size multiplier, blood color, etc.

**Feature purpose**
That way you can instead of just having yellow blood and not showing where the wound is.

**Comment by DindinYT37** [comment]
Don't you mean All is the same as enabled and none is the same as disabled?

**Comment by Vairoon** [comment]
> Don't you mean All is the same as enabled and none is the same as disabled?

Well, by me saying enabled, I mean it as goreless being enabled, and disabled is for it being disabled. Key word being goreLESS.

**Comment by DindinYT37** [comment]
> > Don't you mean All is the same as enabled and none is the same as disabled?
> 
> Well, by me saying enabled, I mean it as goreless being enabled, and disabled is for it being disabled. Key word being goreLESS.

I can read, but the most recent thing you mentioned was **Gore** not **Goreless** which is why I thought you went by that.

**Comment by Vairoon** [comment]
> > > Don't you mean All is the same as enabled and none is the same as disabled?
> > 
> > 
> > Well, by me saying enabled, I mean it as goreless being enabled, and disabled is for it being disabled. Key word being goreLESS.
> 
> I can read, but the most recent thing you mentioned was **Gore** not **Goreless** which is why I thought you went by that.

Fair point, but at the start I did atleast mention "goreless enabled/disabled".

**Comment by DindinYT37** [comment]
> > I can read, but the most recent thing you mentioned was **Gore** not **Goreless** which is why I thought you went by that.
> 
> Fair point, but at the start I did atleast mention "goreless enabled/disabled".

Yeah, I just misinterpreted it apparently.
